docs/src/cache/tutorial006.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+from corva import Api, Cache, ScheduledDataTimeEvent, scheduled
+
+
+@scheduled
+def scheduled_app(event: ScheduledDataTimeEvent, api: Api, cache: Cache):
+    assert cache.get_all() == {}  # <.>
+
+    cache.set_many(  # <.>
+        data=[
+            ('key', 'value_1'),
+            ('key_with_custom_expiry', 'value_2', 1),  # <.>
+        ]
+    )
+
+    assert cache.get_many(  # <.>
+        keys=[
+            'key',
+            'non-existent-key',  # <.>
+        ]
+    ) == {'key': 'value_1', 'non-existent-key': None}
+
+    assert cache.get_all() == {  # <.>
+        'key': 'value_1',
+        'key_with_custom_expiry': 'value_2',
+    }
+
+    time.sleep(1)  # <.>
```

Comment:
deserves a comment why we are sleeping here

Reply:
This file is used in docs. So docs contain all comments. Here is how it looks
![image](https://user-images.githubusercontent.com/72026031/164491701-eeab74be-8303-4adf-8f05-4255bd60e9dc.png)

Reply:
nice!